src/views/print.c

```diff
@@ -322,6 +324,10 @@ void enter(dt_view_t *self)
 
   // prefetch next few from first selected image on.
   dt_view_filmstrip_prefetch();
+
+  dt_control_set_mouse_over_id(prt->image_id);
+  /* signal that we have a new image for the metadata to load the image of current image when entering the module. */
+  dt_control_signal_raise(darktable.signals, DT_SIGNAL_MOUSE_OVER_IMAGE_CHANGE);
```

Comment:
Indeed, but it was not working because the signal is raised only if the image_id is different that what is already recorded. It was recorded by the darkroom module. New patch in a moment.